#
# Copyright (c) 2019 Intel Coporation.
#
# This work is licensed under the terms of the MIT license.
# For a copy, see <https://opensource.org/licenses/MIT>.
#
float32 tire_friction
float32 damping_rate
float32 steer_angle
bool disable_steering
